src/main/java/net/dv8tion/jda/internal/requests/Requester.java

```diff
@@ -66,16 +68,17 @@
 
     public Requester(JDA api)
     {
-        this(api, api.getAccountType());
+        this(api, ((JDAImpl) api).getAuthorizationConfig());
     }
 
-    public Requester(JDA api, AccountType accountType)
+    public Requester(JDA api, AuthorizationConfig authConfig)
     {
-        if (accountType == null)
-            throw new NullPointerException("Provided accountType was null!");
+        if (authConfig == null)
+            throw new NullPointerException("Provided config was null!");
```

Comment:
These are not equivalent. There are now no checks to ensure that AccountType was provided.

Reply:
These were not intended to be equivalent in the first place, the checks should be done in the configs now. I haven't gotten to that though.